src/core/gosling-embed.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Theme } from '..';
-import { GoslingComponent } from './gosling-component';
+import { GoslingComponent, GoslingComponentApi } from './gosling-component';
 import { GoslingSpec } from './gosling.schema';
 
+const MAX_TRIES = 1000;
```

Comment:
```suggestion
const MAX_TRIES = 120;
```

Maybe we can use a smaller number not to wait too long if something is wrong (e.g., 1 min)?